### ISSUE TITLE
Passing the timeout as a positional argument is deprecated

### ISSUE THREAD

**Issue by kivanio** [ixti/sidekiq-throttled-clone#12]
redis (4.8.0)
 sidekiq (6.5.5)
sidekiq-throttled (0.16.2)

```
Passing the timeout as a positional argument is deprecated, it should be passed as a keyword argument:
2022-08-23T01:39:33.476821+00:00 app[delivery_worker.1]: redis.brpop("queue:sms", "queue:emails", "queue:bankapis", "queue:hooks", "queue:searchkick", timeout: 2)(called from: /app/vendor/bundle/ruby/3.0.0/gems/sidekiq-throttled-0.16.2/lib/sidekiq/throttled/fetch.rb:94:in `block in brpop')
```

**Comment by adipasquale** [comment]
this should be fixed with #13

**Comment by ixti** [comment]
will backport #13 into the 0.1 branch and will cut a patch release later this week
